deploy/.env.ingest.production.local.tpl

```diff
@@ -0,0 +1,104 @@
+<%
```

Comment:
Are you going to remove this?

Reply:
no, why? XD

these are markup used by [esh](https://github.com/jirutka/esh), which preprocesses the template to produce the actual script that is then run

Reply:
Ah sorry I thought this was ignored but now I see it is docker ignore not git ignore!